sdk/storage/azure-storage-blob/tests/test_get_blob.py

```diff
@@ -539,7 +524,7 @@ def test_ranged_get_blob_to_path_invalid_range_parallel(self, **kwargs):
             read_bytes = downloader.readinto(temp_file)
 
             # Assert
-            assert read_bytes == blob_size
+            assert read_bytes == blob_size - 1
```

Comment:
What prompted this change in calculation?

Reply:
So, this is in response to this bug fix on this line:
https://github.com/Azure/azure-sdk-for-python/pull/35740/files#diff-0cdd786bf6bdf7516030c98a16e3b07b2d6e3c489a53fd4dc80b1a70637c23d8R457

This is something that has always been broken and this test was verifying the wrong behavior :). Basically, on a partial read (offset and length specified) `readinto` could return the wrong amount of bytes read. It would actually download the blob correctly, it would just report it read some extra bytes when it did not.

In this case, since the download started at `offset=1`, the read size is actually supposed to be one less than the blob size. Test started to fail after we made the fix so updated the test to be correct.